_includes/docs/0.4.x/cloud-foundry.html

```diff
@@ -40,6 +40,11 @@
         <li><pre>$ grunt build</pre></li>
       </ul>
     </li>
+    <li>Run the Gulp Build task to build the optimized JavaScript and CSS files (If you chose Gulp instead of Grunt)
```

Comment:
@ilanbiala Here it should be one or the other. Since they are deploying to a service, it should be what we think is the most reliable. Currently that is grunt. Now, I'd love to eliminate grunt in 0.5 and have everything run with gulp, and in that case we can switch the documentation over to reflect the new gulp set-up. Having both side by side in a deployment walk through is slightly confusing.